t.mjs

```diff
@@ -0,0 +1,27 @@
+import { test } from 'node:test'
```

Comment:
I believe this file shouldn't be included.

Reply:
> I believe this file shouldn't be included.

Why?